### PR TITLE
SL-3769 Implement image upload functionality

### PR DESCRIPTION
**Pull request by sergioebx**
### Description of Changes
- Implement methods to call Images API endpoint to upload image
- Extract common response validation into `ValidationUtils` class

### Testing
Manual testing

### CODE REVIEW

File: src/main/java/com/echobox/api/linkedin/util/ValidationUtils.java [hunk 1]
```diff
@@ -60,5 +65,28 @@ public static void verifyParameterPresence(String parameterName, Object paramete
...
+  
+  /**
+   * Validate response.
+   *
+   * @param response the response from the API call
+   */
+  public static void validateResponse(WebRequestor.Response response) {
+    // If there was no response error information and this was a 401
+    // error, something weird happened on LinkedIn's end. Assume it is a Oauth error.
+    if (HttpURLConnection.HTTP_UNAUTHORIZED == response.getStatusCode()) {
```
Comment (kennethwong-hc) [inline src/main/java/com/echobox/api/linkedin/util/ValidationUtils.java:77]: Can we use `HttpStatus ` to replace `HttpURLConnection`? as `HttpStatus `has a larger set.

**Review by kennethwong-hc — COMMENTED** [review]
Looks good.
Have you performed a manual test?

**Comment by sergioebx** [comment]
> Looks good. Have you performed a manual test?

I've tested this manually :+1:

**Review by kennethwong-hc — APPROVED** [review]
Thanks!

**Pull request merged**